eng/ignore-links.txt

```diff
@@ -12,3 +12,5 @@ https://docs.microsoft.com/javascript/api/@azure/arm-resources-subscriptions
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/resources-subscriptions/arm-resources-subscriptions
 https://docs.microsoft.com/javascript/api/@azure/arm-templatespecs
 https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/templatespecs/arm-templatespecs
+https://docs.microsoft.com/javascript/api/@azure/arm-quota
```

Comment:
We should avoid adding links to the ignore-links.txt file. See http://aka.ms/azsdk/guideline/links for recommendations. 

It looks like some others have already been added to this file as well, please ensure there is a follow-up issue to remove those from this file.